src/cljs/rems/application_list.cljs

```diff
@@ -58,6 +58,12 @@
                           {:class (when (application-util/form-fields-editable? app)
                                     "text-highlight")}
                           value]})
+           :todo (let [value (localize-todo (:application/todo app))]
+                   {:value value
+                    :td [:td.state
+                         {:class (when (str/includes? (str (:application/todo app)) "-your-")
```

Comment:
Two possible semantics for highlight are: 1) highlight if the current user solely needs to act, 2) highlight if the current user or another user in a similar position needs to act.

Selecting 1) means that waiting-for-your-decision, waiting-for-your-review are highlighted (as it is currently implemented).

Selecting 2) would mean that new-application, resubmitted-application, waiting-for-your-decision, waiting-for-your-review are highlighted, as one of the handlers needs to act on an application, but if does not need to be any particular handler.

I think it would be safer to select 2), as the current "conservative" highlighting would allow every one of the handlers to dismiss a new or re-submitted application if they based their decision to act on highlighting only.

Reply:
Highlighting for the handler has been added.